lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -62,14 +62,13 @@ class UnusedBytes extends Audit {
    * @return {!Promise<!AuditResult>}
    */
   static audit(artifacts) {
-    const devtoolsLogs = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
-    return artifacts.requestNetworkRecords(devtoolsLogs).then(networkRecords => {
-      return artifacts.requestNetworkThroughput(networkRecords).then(networkThroughput =>
-        Promise.resolve(this.audit_(artifacts, networkRecords)).then(result =>
-          this.createAuditResult(result, networkThroughput)
-        )
-      );
-    });
+    const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    return artifacts.requestNetworkRecords(devtoolsLog)
+      .then(networkRecords => this.audit_(artifacts, networkRecords))
+      .then(result => {
```

Comment:
>  just trying to remove nesting?

yeah, and it gets rid of the `Promise.resolve`. I tried a few different permutations and this was a less awkward version than others, but I can revert to

```js
return artifacts.requestNetworkRecords(devtoolsLogs).then(networkRecords => {
  return artifacts.requestNetworkThroughput(devtoolsLogs).then(networkThroughput =>
    Promise.resolve(this.audit_(artifacts, networkRecords)).then(result =>
      this.createAuditResult(result, networkThroughput)
    )
  );
});
```
if you want.